src/Init.lean

```diff
@@ -15,6 +15,7 @@ public import Init.RCases
 public import Init.Core
 public import Init.Control
 public import Init.WF
+public import Init.WFC
```

Comment:
`WFComputable` maybe? No need to be saving characters here

Reply:
I think that's a better name.  I've done the renamings.